admin/templates/admin/admin.html

```diff
@@ -13,109 +13,105 @@
+                            <button type="button" id="admin-create-preview" class="btn btn-secondary ms-2">Preview
+                                Terms</button>
+                        </form>
                     </div>
-                    <button type="submit" class="btn btn-primary" name="Change Theme">Change Theme</button>
-                </form>
-            </div>
-        </section>
-
-        <!-- Update Publication Terms -->
-        <section class="card my-3">
-            <div class="card-body">
-                <h2 class="card-title">Update Publication Terms</h2>
-                <div class="d-flex justify-content-start align-items-end" style="width: 100%;">
-                    <!-- Form to update publication terms -->
-                    <form action="{{ url_for('admin_bp.set_publication_terms') }}" method="POST"
-                        class="flex-fill me-2 needs-validation" novalidate>
-                        <input type="hidden" name="csrf_token" value="{{ csrf_token() }}">
-                        <div class="mb-3">
-                            <label for="publicationTerms">Current Publication Terms:</label>
-                            <textarea class="form-control" id="publicationTerms" name="publicationTerms" rows="10"
-                                style="width: 110%;" required>{{ publication_terms|safe }}</textarea>
-                        </div>
-                        <button type="submit" id="update-terms" class="btn btn-primary">Update Terms</button>
-                    </form>
-                    <!-- Form to preview updated terms -->
-                    <button type="button" id="create-preview" class="btn btn-secondary">Preview Terms</button>
                 </div>
             </div>
         </section>
+    </main>
+</div>
 
-        <!-- Confirmation Agreement Conditions Modal, copied from vault/browse.html -->
-        <div class="modal" tabindex="-1" role="dialog" id="confirmAgreementConditions">
-            <div class="modal-dialog">
-                <div class="modal-content">
-                    <div class="modal-body">
-                        <!-- Updated content will be displayed here -->
+<!-- Confirmation Agreement Conditions Modal, copied from vault/browse.html -->
+<div class="modal" tabindex="-1" role="dialog" id="confirmAgreementConditions">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-body">
+                <!-- Updated content will be displayed here -->
+            </div>
+            <div class="modal-footer">
+                <div class="row">
+                    <div class="col-sm-12">
+                        <fieldset>
+                            <input type="checkbox" class="confirm-conditions" id="checkbox-confirm-conditions">
+                            <label for="checkbox-confirm-conditions">Please confirm that you agree with the
+                                above</label>
+                        </fieldset>
```

Comment:
Maybe you can extract this modal to a template file that we can `include` both in browse.html in the vault and here. But if it turns out to be too much work this is good as well.

Reply:
This is a nice suggestion. I'll look it up

Reply:
Nice, can you make the Confirm button dismiss the modal but only when in the admin page?